pkg/e2e/start_stop_test.go

```diff
@@ -229,22 +229,35 @@ func TestStartStopMultipleServices(t *testing.T) {
 		cli.RunDockerComposeCmd(t, "down", "--remove-orphans", "-v", "-t", "0")
 	})
 
-	cli.RunDockerComposeCmd(t, "up", "-d", "--wait")
+	t.Run("starts/stops multiple services", func(t *testing.T) {
```

Comment:
Easier to look at the diff here with whitespace off, I just wrapped up a previous test in a `t.Run(...` and added a separate one